README.md

```diff
@@ -165,6 +168,29 @@ To switch from the default in-memory SQLite database to a file-based SQLite data
 
 2.  Edit your Run Configuration via **Run** > **Run Configurations...** and select enter the **Profile** `sqlite` on tab **Spring** and click **Run**.
 
+## Deploy to SAP Cloud Platform
+
+CAP Java applications can be deployed to the SAP Cloud Platform either in single tenant or in multitenancy mode (see [Multitenancy in CAP Java](https://cap.cloud.sap/docs/java/multitenancy) for more information.
+
+Prerequisites:
+- Install the [Cloud MTA Build Tool](https://sap.github.io/cloud-mta-build-tool/): `npm install -g mbt
+- Install the [Cloud Foundry Command Line Interface](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+- Get a Cloud Platform account to deploy the services and applications
+
+Deploy as Single Tenant Application
+- Rename `mta-st.yaml` to `mta.yaml`
+- Run `mbt build`
+- Run `cf login`
+- Run `cf deploy mta_archives/bookshop-java-public_1.0.0.mtar`
+
+Deploy as Multitenant Application:
+- Create a HANA Cloud Instance in your Cloud Platform space
+- Run `mbt build`
```

Comment:
What happened to renaming mta-mt.yaml to mta-yaml? Is that not required any more?